baseapp/baseapp.go

```diff
@@ -37,7 +38,7 @@ import (
 )
 
 type (
-	execMode uint8
+	execMode = state.ExecMode
```

Comment:
can't we just use the ExecMode type from `types` everywhere? never really understood why this has so many aliases

Reply:
no we should use types you're right